src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const basicDayCost = 40;
+  const fullCost = days * basicDayCost;
+  const regularDiscount = 20;
+  const weekDiscount = 50;
+
+  const regularMinDays = 3;
+  const weekMinDays = 7;
+
+  if (days < regularMinDays) {
+    return fullCost;
+  } else if (days < weekMinDays) {
+    return fullCost - regularDiscount;
+  }
```

Comment:
I would use "if" condition several times than "else if" for better readability. but it's ok